.github/workflows/build-action.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   Build-And-Test-Server:
-    timeout-minutes: 180
+    timeout-minutes: 210
```

Comment:
CI now takes up to 3.5hrs with the remaining DEX tests :( ill try to fix that while I am working on the other CI issue